cfg/rippled-example.cfg

```diff
@@ -818,13 +818,11 @@
 #       RocksDB is an alternative backend for systems that don't use solid-state
 #       drives.  Because RocksDB's performance degrades as it stores more data,
 #       keeping full history is not advised, and using online delete is
-#       recommended.  RocksDB is not available on Windows.
+#       recommended.
 #
 #       The RocksDB backend also provides these optional parameters:
```

Comment:
```suggestion
#       The RocksDB backend provides no optional parameters
```

Reply:
...except that the rocks backend *does* have a number of optional configs but we have chosen (so far) to not document them because they are considered "expert only" 😬 .

Reply:
@MarkusTeufelberger sorry - I initially misunderstood your comment. This is confusing and I will remove the sentence entirely (I'll say nothing about optional config params for now). thanks.

Reply:
Yeah, that's probably even better, RocksDB was/is the odd one out regarding optional parameters anyways.

However, at the end of the file quite a lot of existing undocumented optional parameters for RocksDB are set and used...

Reply:
I agree..it's a little off-putting that we use some options here that we don't document. We might need to address that in the future (for now, all rocks specific options are considered "expert only")